main/views.py

```diff
@@ -1,57 +1,137 @@
+import json
+import datetime
+
+
 from django.http import HttpResponse, JsonResponse
 from django.shortcuts import get_object_or_404, render
-
 from .models import *
 
 
 def index(request):
+    # i will remove matches and use results instead
```

Comment:
I'd like you to rethink/rewrite this view. I think the most useful format for returning the results would be:
```Python
{'matches': [
	{
		'id': ...,
		'date': ...,
		'winner': {
			'id': ...,
			'name': ...,
			'score': ...,
		},
		'loser': {
			'id': ...,
			'name': ...,
			'score': ...,
		}
	}
]}
```